src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -168,7 +168,7 @@ internal ImmutableList<RoleName> Roles
     /// `AskTimeoutException: sending to terminated ref breaks promises`. Using lazy
     /// val is fine.
     /// </summary>
-    public abstract class MultiNodeSpec : TestKitBase, IMultiNodeSpecCallbacks
+    public abstract class MultiNodeSpec : TestKitBase, IMultiNodeSpecCallbacks, IDisposable
```

Comment:
Important change: making the `MultiNodeSpec` `IDisposable` will allow XUnit to call the `Dispose` method after the test is finished, and we need specific termination behavior in order to eliminate races in the MNTR.